clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -5151,6 +5151,14 @@ bool TokenAnnotator::mustBreakBefore(const AnnotatedLine &Line,
     return true;
   if (Left.IsUnterminatedLiteral)
     return true;
+  if (Right.is(tok::lessless) && Right.Next && Left.is(tok::string_literal) &&
+      // FIXME: Breaking after newlines seems useful in general. Turn this into
+      // an option and Recognize more cases like endl etc, and break independent
+      // of what comes after operator lessless.
+      Right.Next->is(tok::string_literal) &&
+      Left.TokenText.ends_with("\\n\"")) {
```

Comment:
```suggestion
  if (Left.is(tok::string_literal) && Left.TokenText.ends_with("\\n\"") && Right.is(tok::lessless) &&
      Right.Next && Right.Next->is(tok::string_literal) ) {
      // FIXME: Breaking after newlines seems useful in general. Turn this into
      // an option and Recognize more cases like endl etc, and break independent
      // of what comes after operator lessless.
```

Or similar, right now it is a bit unsorted. (This is obviously not formatted.)